Sources/ClientRuntime/Networking/Http/Middlewares/ContentLengthMiddleware.swift

```diff
@@ -40,7 +40,7 @@ public struct ContentLengthMiddleware<OperationStackOutput>: Middleware {
                 // Transfer-Encoding can be sent on all Event Streams where length cannot be determined
                 // or on blob Data Streams where requiresLength is true and unsignedPayload is false
                 // Only for HTTP/1.1 requests, will be removed in all HTTP/2 requests
-                input.headers.update(name: "Transfer-Encoding", value: "Chunked")
+                input.headers.update(name: "Transfer-Encoding", value: "chunked")
```

Comment:
what's the correct capitalization for the header name? 'Transfer-Encoding', 'Transfer-encoding', or 'transfer-encoding'?

Reply:
HTTP header names are case-insensitive, but the values are not.
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers

The value for `Transfer-Encoding` should be lowercase `chunked`:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Transfer-Encoding